Doc/library/logging.rst

```diff
@@ -397,21 +397,40 @@ have specific values relative to the predefined levels. If you define a level
 with the same numeric value, it overwrites the predefined value; the predefined
 name is lost.
 
-+-----------------------+---------------+
-| Level                 | Numeric value |
-+=======================+===============+
-| .. py:data:: CRITICAL | 50            |
-+-----------------------+---------------+
-| .. py:data:: ERROR    | 40            |
-+-----------------------+---------------+
-| .. py:data:: WARNING  | 30            |
-+-----------------------+---------------+
-| .. py:data:: INFO     | 20            |
-+-----------------------+---------------+
-| .. py:data:: DEBUG    | 10            |
-+-----------------------+---------------+
-| .. py:data:: NOTSET   | 0             |
-+-----------------------+---------------+
++-----------------------+---------------+-------------------------------------+
+| Level                 | Numeric value | What it means / When to use it      |
++=======================+===============+=====================================+
+| .. py:data:: NOTSET   | 0             | When set on a logger, indicates that|
+|                       |               | ancestor loggers are to be consulted|
+|                       |               | to determine the effective level. If|
+|                       |               | If that still resolves to           |
+|                       |               | :const:`NOTSET`, then all events are|
+|                       |               | logged. When set on a handler, all  |
+|                       |               | events are handled.                 |
```

Comment:
The markup for NOTSET here would cause it to add a link which would just take you to where you already are, which would be annoying. Better to use `!` to suppress the link:

```suggestion
|                       |               | :const:`!NOTSET`, then all events   |
|                       |               | are logged. When set on a handler,  |
|                       |               | all events are handled.             |
```